src/plugins/appeals/api.ts

```diff
@@ -57,6 +57,10 @@ app.post("/ban-appeal", async (req, res) => {
                 },
             );
         const actionRow = new ActionRowBuilder().addComponents(
+            new ButtonBuilder()
+                .setLabel("Thread")
+                .setStyle(ButtonStyle.Secondary)
+                .setCustomId(`appeal-thread-${tag}-${id}`),
```

Comment:
This violates the button custom ID principle of `<type>-<action>-<target-id>` that is observed in unbanAppeal.ts and santaSquad.command.ts

Reply:
Changed to keep consistent principle. For now, the thread will use the userId in order to differentiate posts, will need a solution that’s user friendly to read.